lib/rules/no-private-routing-service.js

```diff
@@ -33,6 +33,9 @@ module.exports = {
       Property(node) {
         if (
           emberUtils.isInjectedServiceProp(node) &&
+          node.value &&
+          node.value.arguments &&
```

Comment:
Instead of using null checks, we should actually be using type checks:

```suggestion
          types.isCallExpression(node.value) &&
```

A `Property` will always have a `value` and a `CallExpression` will always have an `arguments` array (although it could be empty so that check you added is important).

Reply:
Actually I think that `isInjectedServiceProp` covers the case of `{ aProp: '' }`, so I think I can just check whether args length is truthy